src/sendMetaTransaction.ts

```diff
@@ -95,20 +96,41 @@ export async function sendMetaTransaction(
       }
     )
 
-    if (!res.ok) {
-      throw new Error(res.statusText)
+    const { ok, txHash, message } = (await res.json()) as {
```

Comment:
should we wrap this with a try/catch jn case the response is not a valid json (ie some 5XX error)?

Reply:
Thought about it! but the entire method is inside a trycatch and tries to handle all errors. A 500 will result in a `throw new MetaTransactionError(error.message, ErrorCode.UNKNOWN)` I believe, right?

Reply:
ahh cool then 👌🏻